plugins/modules/host_contain.py

```diff
@@ -70,10 +70,14 @@
       - "09876543210987654321"
     contained: no
 
-- name: Contain all hosts in RFM
+- name: Contain all Linux hosts in RFM (using host_ids lookup)
   crowdstrike.falcon.host_contain:
-    hosts: "{{ lookup('crowdstrike.falcon.host_ids', 'reduced_functionality_mode:\"yes\"') }}"
+    hosts: "{{ lookup('crowdstrike.falcon.host_ids', contain_filter) }}"
     contained: yes
+  vars:
+    contain_filter: >
+      platform_name:'Linux' +
+      reduced_functionality_mode:'yes'
```

Comment:
I think `linux_rfm_filter` would make more sense/be more consistent with your other examples. That, or just call it `filter` for all task-level var examples since it doesn't really matter what it's called.